ODataConnectedService/src/Common/CodeGeneratorUtils.cs

```diff
@@ -31,7 +31,7 @@ public static string GetWCFDSInstallLocation()
                 }
             }
 
-            return null;
+            return String.Empty;
```

Comment:
Thanks for the fix! Before we merge, can you please update the comment at line 16? It should say that we're returning String.Empty instead of null with your update.